src/inject.js

```diff
@@ -154,13 +160,15 @@ export const injectStyleOnce = (
     useImportant /* : boolean */,
     selectorHandlers /* : SelectorHandler[] */ = []
 ) => {
-    if (!alreadyInjected[key]) {
-        const generated = generateCSS(
-            selector, definitions, selectorHandlers,
-            stringHandlers, useImportant);
-
-        injectGeneratedCSSOnce(key, generated);
+    if (alreadyInjected[key]) {
```

Comment:
I doubt it--I think I just did this out of habit as I find the early return pattern to be generally more readable. Want me to revert this back?